rke/k8s_rke_system_images.go

```diff
@@ -7890,6 +7890,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		// Enabled out of band post v2.7.0
+		"v1.22.17-rancher1-2": {
```

Comment:
`v1.22.17-rancher1-1` is not yet in the release branch so it's perfectly fine to update images in `v1.22.17-rancher1-1` instead of introducing `v1.22.17-rancher1-2` - no reason I can think of to introduce more than one version per minor k8s version in single KDM update.

The same applies to `v1.23.15-rancher1-1` and `v1.24.9-rancher1-1`.

Reply:
Resolved in [afe75d3](https://github.com/rancher/kontainer-driver-metadata/pull/1044/commits/afe75d357ed380cc14b9c708494309cc54fd52dc)